docs/content/2.frontier/6.templates/1.dotNet.md

```diff
@@ -1,2 +1,40 @@
-# [ Backend ] - Dot Net
+# [ Backend ] - .Net
 
+This template provieds a multi-project solution for creating an enterprise-level application that includes the use of .NET Identity, Swagger, API Versioning, CORS, Localization, NLog, Fluent Validation, Fluent Email, IP Rate Limiting using the .NET 5 Framework and are guided by the Clean Architecture principles enabling rapid application development.
+
+<br />
+
+## Why use .NET
+___
+1. **Security** -  .NET provides highly secured environment through the usage of various security mechanisms to keep an application protected. Through the provision of many useful classes and services, .NET enables developers to use cryptography, write more secure code, and implement role-based security. The security support provided by .NET allows for the flexibility and extensibility to meet the needs of a wide range of applications.
```

Comment:
Subtle again lol but maybe instead of ".NET provides highly secured environment" you can say:
- ".NET provides **a** highly secured environment"
- ".NET provides highly secured environment**s**"